lib/auth/auth.go

```diff
@@ -5707,6 +5720,111 @@ func (a *Server) syncDynamicLabelsAlert(ctx context.Context) {
 	}
 }
 
+func (a *Server) cleanupNotifications(ctx context.Context) {
+	var userNotifications []*notificationsv1.Notification
+	var userNotificationsPageKey string
+	userNotificationsTicker := time.NewTicker(notificationsPageReadInterval)
```

Comment:
nit: consider using names for the tickers that communicate the fact that they are being used as a rate limiting device.  e.g. this could be something like `userNotificationReadLimiter`.